integration_tests/integration/notifications-settings.spec.ts

```diff
@@ -30,6 +30,32 @@ context('A staff member can view their notification settings', () => {
     Page.verifyOnPage(NotificationManagePage)
 
     cy.contains('You receive notifications')
+    cy.contains('Pause notifications').click()
+
+    cy.contains('How long do you want to pause notifications for')
```

Comment:
this style looks to be different from the rest in that you haven't added a new page object and interact with the page object.  Instead looks to be all done here, esp. interacting with inputs, selects and buttons.